source/includes/steps-stop-sharded-cluster.yaml

```diff
@@ -0,0 +1,34 @@
+title: "Stop :binary:`mongos <bin.mongos>` routers."
```

Comment:
Should there be a pre-req to stop the balancer and wait for chunk migrations to complete?

Reply:
done